plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/component/DigitalExchangeComponentListProcessor.java

```diff
@@ -71,18 +73,18 @@ public DigitalExchangeComponentListProcessor(RestListRequest restListRequest, St
         return sort -> {
             switch (sort) {
                 case LAST_UPDATE:
-                    return (a, b) -> a.getLastUpdate().compareTo(b.getLastUpdate());
+                    return (a, b) -> ObjectUtils.compare(a.getLastUpdate(), b.getLastUpdate());
                 case VERSION:
-                    return (a, b) -> a.getVersion().compareToIgnoreCase(b.getVersion());
+                    return (a, b) -> StringUtils.compare(a.getVersion(), b.getVersion());
```

Comment:
Why the version requires exact match and not ignore case?

Reply:
Thanks, it's a mistake.